vitest.config.mts

```diff
@@ -14,7 +14,7 @@ export default defineConfig({
     coverage: {
       provider: 'v8',
       include: ['lib/**/*.ts'],
-      exclude: ['lib/**/*.spec.ts', 'lib/types.ts', 'lib/index.ts', 'lib/errors/publicErrors.ts'],
+      exclude: ['lib/**/*.spec.ts', 'lib/types.ts', 'lib/index.ts', 'lib/errors/publicErrors.ts', 'lib/plugins/bullMqMetricsPlugin.ts', 'lib/plugins/bull-mq-metrics/MetricsCollector.ts', 'lib/plugins/bull-mq-metrics/queueDiscoverers.ts'],
```

Comment:
probably exclusion no longer needed?

Reply:
Wait, I haven't covered everything yet 😅 
This still needs some work